src/Stripe.net/Services/Tokens/TokenPiiOptions.cs

```diff
@@ -0,0 +1,13 @@
+namespace Stripe
+{
+    using Newtonsoft.Json;
+
+    public class TokenPiiOptions : INestedOptions
+    {
+        /// <summary>
+        /// The <c>IdNumber</c> for the PII, in string form.
```

Comment:
why the `<c>` tags here? Shouldn't this just be `The personal id number for the PII` or similar? Feels like this is trying to reference another property elsewhere. Not blocking though

Reply:
It was `id_number` so I pascal cased it, but again I'm not sure if we'll be able to do this with autogen anyways